ultravox/tools/ds_tool/ds_tool.py

```diff
@@ -23,6 +26,8 @@ class TtsTask:
     audio_column_name: Optional[str] = simple_parsing.field(default=None, alias="-a")
     voice: Optional[str] = simple_parsing.field(default=None, alias="-V")
     sample_rate: int = simple_parsing.field(default=16000, alias="-r")
+    write_batch_size: int = 1000
```

Comment:
suggest adding this as an argument to `DatasetToolArgs` since it's common across tasks, similar to `num_workers`.